### ISSUE TITLE
explore MetalLB

### ISSUE THREAD

**Issue by billimek** [billimek/k8s-templates#66]
See:

* https://metallb.universe.tf/
* https://medium.com/@ipuustin/using-metallb-as-kubernetes-load-balancer-with-ubiquiti-edgerouter-7ff680e9dca3
* https://hackernoon.com/metallb-a-load-balancer-for-bare-metal-kubernetes-clusters-f7320fde52f2

**Comment by billimek** [comment]
See also https://medium.com/@carlosedp/multiple-traefik-ingresses-with-letsencrypt-https-certificates-on-kubernetes-b590550280cf

**Comment by billimek** [comment]
I got it to work, sort of.

Did the BGP implementation and saw that traefik was getting assigned an IP.  I didn't have luck getting the router to forward to the new ip though.

As I looked into this more, I don't see a big value-proposition here.

As things stand, I have a good haproxy setup with traefik exposed via NodePort.  Metallb buys me an automatic IP assignment, but I already have a static IP I want to use.  It actually seems like it will be more work to go down the metallb route.

Closing this exploration issue for now.  May revisit some day.
